src/theme/widget.ts

```diff
@@ -1,5 +1,12 @@
 import { colorsDark, colorsLight } from './colors'
 
+const fonts = {
+  fontFamily: {
```

Comment:
If they are the same, this should be simplified to just use a string:

    const fontFamily = 'Inter custom'

The widget will check for `typeof fonts === 'string'` and use the value directly.